ui/apps/platform/src/Components/SecurityContext.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { isEmpty } from 'lodash';
+import {
+    Card,
+    CardBody,
+    CardTitle,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    EmptyState,
+} from '@patternfly/react-core';
+
+import { ContainerSecurityContext } from 'types/deployment.proto';
+
+type SecurityContextProps = {
+    securityContext: ContainerSecurityContext;
+};
+
+function SecurityContext({ securityContext }: SecurityContextProps) {
+    // build a map of only those properties that actually have values
+
+    // sort the keys of the prop, so that the map is in alpha order
+    const sortedKeys = Object.keys(securityContext).sort();
+
+    //
+    const filteredValues = new Map();
+    sortedKeys.forEach((key) => {
```

Comment:
Proposal: This logic might benefit from being extracted into a util function + tested